exercises/J-array-get-set/exercises2.js

```diff
@@ -7,7 +7,8 @@
 */
 
 var numbers = [1, 2, 3]; // Don't change this array literal declaration
-
+numbers[3] = 4;
+numbers[0] = "?";
```

Comment:
Why "?" it should be 1

Reply:
1 is already in the array.
Hence the '?'.
I could've left it at 1